src/models/apis/leaflet.ts

```diff
@@ -30,7 +30,7 @@ export default class Leaflet implements IMapFunctions {
 
     constructor() { /* */ }
 
-    public initialize(mapType: MapType, params: any, elementId: string) {
+    public initialize(mapType: MapType, params: any, elementId?: string): Promise<any> {
```

Comment:
mesma situação do google maps
```suggestion
    public initialize(mapType: MapType, params: any, elementId: 'inlog-map'): Promise<any> {
```

Reply:
Alteração adicionada